build/ps-entry.sh

```diff
@@ -480,6 +480,11 @@ if [[ $originalArgOne == mongo* ]]; then
 		_mongod_hack_rename_arg_save_val --sslDisabledProtocols --tlsDisabledProtocols "${mongodHackedArgs[@]}"
 	fi
 
+	if [[ $originalArgOne == "mongod" ]]; then
+		_mongod_hack_ensure_arg_val --logpath "/data/logs/mongod.log" "${mongodHackedArgs[@]}"
+		_mongod_hack_ensure_arg --logappend "${mongodHackedArgs[@]}"
+	fi
```

Comment:
should we do this by default or only if logcollector is enabled

Reply:
good point, we should only add them if logcollector is enabled.

Reply:
Introduced a new env var which is set if log collector is enabled and it is checked here - `LOGCOLLECTOR_ENABLED`.